Documentation/Releases/solr-release-7-0.rst

```diff
@@ -17,29 +17,335 @@
+This is useful, when you setup multiple instances with own entry templates.
+
+* https://github.com/TYPO3-Solr/ext-solr/pull/1325
+
+Refactoring of Query API
+------------------------
+
+The Query class is one of the biggest classes in EXT:solr and grown over time. This class has now been splitted into several classes.
+Along with that a concept of "ParameterBuilder" has been introduced. A ParameterBuilder is responsible to build a parameter part of the query.
+E.g. the Grouping ParameterBuilder is responsible to build all parameters of the solr query for the grouping.
+
+* https://github.com/TYPO3-Solr/ext-solr/pull/1385
+
+Move FilterEncoder and FacetBuilder to Facet Package
+----------------------------------------------------
+
+In Solrfluid there was one folder for each facet, that contains the facet class and a parser that parsers the solr response into the facet object.
+The opposite part(parse the url, build the solr query) was previously done in EXT:solr, with a FilterEncoder that was registered in the FacetRendererFactory.
+
+Now because solrfluid and solr have been merged, this logic can also be streamlined. Every facet is now structured in a FacetPackage.
+
+A FacetPackage describes:
+
+* Which parser should be used to parse the solr response
+* Which url decoder should be used to parse the EXT:solr query data
+* Which query builder should be used to build the faceting query part
+
+You can also implement custom facet types be registering an on FacetPackage with the FacetRegistry.
+
+**Migration**:
+
+When you have implemented an own FacetParser for solrfluid, you should add a FacetPackage, that references a UrlDecoder and QueryBuilder.
+If you have used a custom FacetParser without registring a custom facet type in EXT:solr (ApacheSolrForTypo3\Solr\Facet\FacetRendererFactory::registerFacetType) you can just reference DefaultUrlDecorder and DefaultFacerQueryBuilder in you FacetPackage.
+
+* https://github.com/TYPO3-Solr/ext-solr/pull/1319
+
+Custom plugin namespace - Multiple Instances
+--------------------------------------------
+
+Before solrfluid was merged there were several parts in EXT:solr where the data was read using GeneralUtility::_GET. The drawback of this approach is that the structure of the urls is hard to change and it is not possible to have custom namespaces for each instance of a pluigin.
+
+With solrfuid a SearchRequest object was introduced. This object holds all data from the user request. Now this object is used, whenever data from the user action is read. This allows us to make the request namespace changeable. You can now add you custom plugin namespace to a search plugin instance.
+
+* https://github.com/TYPO3-Solr/ext-solr/pull/1379
+
+Doctrine Migration
+------------------
+
+As an ongoing task, we started with the migration of database queries to doctrine. Since the database is used in many parts of the extension there a still many parts open.
```

Comment:
- doctrine => Doctrine
- "... the extension there **are** still many parts open."